src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -41,6 +41,14 @@ configurationRegistry.registerConfiguration({
 		'terminal.integrated.fontFamily': {
 			'description': nls.localize('terminal.integrated.fontFamily', "The font family used by the terminal (CSS font-family format), this defaults to editor.fontFamily's value."),
 			'type': 'string'
+		},
+		'terminal.integrated.fontSize': {
+			'description': nls.localize('terminal.integrated.fontSize', "The font size used by the terminal (in pixels), this defaults to editor.fontSize's value."),
```

Comment:
Sorry but let's go for consistency with the editor settings for this in commonEditorConfig.ts:

```
"Controls the font family of the terminal."
"Controls the font size of the terminal."
"Controls the line height of the terminal."
```